store/cachekv/benchmark_test.go

```diff
@@ -8,41 +8,51 @@ import (
 	"github.com/cosmos/cosmos-sdk/store"
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
a bit out of scope but any chance you want to refactor this benchmark to not import sdk?this is the final import of the sdk in order to spin store into its own go.mdo

Reply:
maybe easier to move it out from store, maybe by the side of context.

Reply:
done https://github.com/cosmos/cosmos-sdk/pull/14456